engine/runtime/src/main/java/org/enso/interpreter/service/ExecutionService.java

```diff
@@ -4,25 +4,10 @@
 import com.oracle.truffle.api.TruffleLogger;
 import com.oracle.truffle.api.instrumentation.EventBinding;
 import com.oracle.truffle.api.instrumentation.ExecutionEventListener;
-import com.oracle.truffle.api.interop.ArityException;
-import com.oracle.truffle.api.interop.InteropLibrary;
-import com.oracle.truffle.api.interop.UnknownIdentifierException;
-import com.oracle.truffle.api.interop.UnsupportedMessageException;
-import com.oracle.truffle.api.interop.UnsupportedTypeException;
+import com.oracle.truffle.api.interop.*;
```

Comment:
I'd rather avoid using `*` imports. Would it be possible to configure our `sbt javafmt` to reject star imports, @hubertp?

Reply:
google-java-formatter (called by javafmt behind the scenes) does not really support that and it is out-of-scope IIRC